library/src/lib/inline-help/inline-help.component.ts

```diff
@@ -1,29 +1,35 @@
-import { Component, Input, ViewEncapsulation } from '@angular/core';
+import { Component, ElementRef, Input, OnChanges, SimpleChanges, ViewEncapsulation } from '@angular/core';
+import { Placement } from 'popper.js';
+import { AbstractFdNgxClass } from '../utils/abstract-fd-ngx-class';
 /**
  * The component that represents an inline-help. 
  * Inline help is used to display help text in a popover, often inline with headers, body text and form labels.
  *
  * ```html
- * <fd-inline-help [position]="'bottom-left'">
+ * <fd-inline-help [placement]="'bottom-left'">
  *      Lorem ipsum dolor sit amet, consectetur adipiscing.
  * </fd-inline-help>
  * ```
  */
 @Component({
     selector: 'fd-inline-help',
     templateUrl: './inline-help.component.html',
-    host: {
-        class: 'fd-inline-help',
```

Comment:
A very minor detail, but because this class is no longer applied, the text of the inline help is grey and not black as designed in fundamental-styles

Reply:
Generally this class cannot be applied to parent, because it has some css styling that will break component inside popover. I had to interfere styling, what is not best thing I could do, because of possible future problems with following any fiori styles changes. What do you think about it @MattL75 ?

Reply:
I don't think there's a clear way around this. We can't even go and remove it from Fundamental Styles because the component needs to work without popper.js. I suppose one option would be to include a 'hidden' class in Styles which wouldn't incorporate the style-popovers and would let us develop our own. As a workaround, maybe you can add the same text color variable to your custom scss?